content/practices/pre-ipm/index.md

```diff
@@ -0,0 +1,84 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+
+title: "Pre-IPM"
```

Comment:
I feel we need to spell out the title. Pre-Iteration Planning Meeting or similar.